web/content/js/nav.js

```diff
@@ -23,18 +23,6 @@ function buildNavigation() {
 </button>
 <ul class="nav-menu">
   <li class="nav-item"><a href="https://theforeman.org/">About Foreman</a></li>`
-    + navBuilds.map(function(build){
```

Comment:
Now that you no longer need `navBuilds` you can also remove the `const navBuilds` declaration.

Reply:
Can I really? Because it causes the top menu bar to disappear.